components/archiving/archiveDao.js

```diff
@@ -0,0 +1,467 @@
+'use strict';
+
+const { db, pgp, NotFoundError } = require('../../utils/dbUtils');
+const logger = require('../../utils/logger').logger;
+const queryStream = require('pg-query-stream');
+const qrec = pgp.errors.queryResultErrorCode;
+
+// archiveEntity info
+const id = 'id';
+const created = 'created';
+const modified = 'modified';
+const archiveEntityId = 'archive_entity_id';
+const archiveType = 'entity_type';
+const entityIdentifierColumn = 'entity_identifier';
```

Comment:
"XId(entifier)Column" makes me think that it's the primary key column on the table, but that's the term you've used for the case ID/project name column.
Can we disambiguate this - something like `archiveEntityPkColumn` for the table ID, and `entityIdentifier` for the case ID/project name column, or something else?